build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# This script is triggered by the json manifest.
```

Comment:
this script is unecessary, overly complex and should instead be 6 lines of `build-commands`.

Reply:
Also, now that I've moved the icon, desktop and appstream files to upstream, do I need to include the git repository instead of the files in the sources?

Reply:
Deleted in https://github.com/flathub/flathub/pull/3939/commits/588131b7afc7f48d8890d57c3b50347a05d9bfb1

Rewritten in https://github.com/flathub/flathub/pull/3939/commits/220acf01825e23d4ec393962985e2ac5a2d48b0e